src/Development/Rattle/Server.hs

```diff
@@ -230,6 +255,29 @@ cmdRattleRun rattle@Rattle{..} cmd@(Cmd opts args) startTimestamp hist msgs = do
                     let f hasher xs = mapMaybeM (\x -> fmap (\(mt,h) -> (x,mt,h)) <$> hasher x) $ filter (not . skip) xs
                     touch <- Touch <$> f hashFileForward (tRead touch) <*> f hashFile (tWrite touch)
                     touch <- generateHashForwards cmd [x | HashNonDeterministic xs <- opts2, x <- xs] touch
+                    when (isJust $ rattleDebug options) $
```

Comment:
Better to use whenJust, which gives you the thing inside the just, and let syou avoid the fromJust